test/testSamples/MaterialX/MtlxUVStreamTest.usda

```diff
@@ -0,0 +1,107 @@
+#usda 1.0
+(
```

Comment:
If the "HDVP2_DEBUG_MATERIAL" TF_DEBUG flag is set, you will see the corrected MaterialX document printed in the output:
```
<?xml version="1.0"?>
<materialx version="1.38" xmlns:xi="http://www.w3.org/2001/XInclude">
  <standard_surface name="SR_USD_Mtlx_VP2_Material" type="surfaceshader">
    <input name="base_color" type="color3" output="base_color_out" nodegraph="NG_Maya" />
  </standard_surface>
  <surfacematerial name="USD_Mtlx_VP2_Material" type="material">
    <input name="surfaceshader" type="surfaceshader" nodename="SR_USD_Mtlx_VP2_Material" />
  </surfacematerial>
  <nodegraph name="NG_Maya">
    <mix name="N1" type="color3">
      <input name="bg" type="color3" nodename="N6" />
      <input name="fg" type="color3" nodename="N5" />
      <input name="mix" type="float" nodename="N2" />
    </mix>
    <swizzle name="N6" type="color3">
      <input name="channels" type="string" value="rgb" />
      <input name="in" type="color4" nodename="N7" />
    </swizzle>
    <image name="N7" type="color4">
      <input name="texcoord" type="vector2" defaultgeomprop="UV0" uiname="Texture Coordinates" nodename="ST_reader" />
    </image>
    <ramptb name="N5" type="color3">
      <input name="texcoord" type="vector2" defaultgeomprop="UV0" nodename="ST_reader" />
    </ramptb>
    <add name="N2" type="float">
      <input name="in1" type="float" nodename="N3" />
    </add>
    <noise2d name="N3" type="float">
      <input name="texcoord" type="vector2" nodename="N4" />
    </noise2d>
    <output name="base_color_out" type="color3" nodename="N1" />
    <geompropvalue name="ST_reader" type="vector2">
      <input name="geomprop" type="string" value="st" uniform="true" />
    </geompropvalue>
    <geompropvalue name="N4" type="vector2">
      <input name="geomprop" type="string" value="st1" uniform="true" />
    </geompropvalue>
  </nodegraph>
</materialx>
```